### PR TITLE
D8CORE-1409: Enable new filter in full html editor format.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW 

# Summary
- Enables new filter in full html editor format that we added to stanford_media
- Updates module with changes from core profile config.

# Needed By (Date)
- Thursday Please

# Urgency
- Med/low

# Steps to Test

1. Review code
2. TBD.

### CODE REVIEW

File: config/install/filter.format.stanford_html.yml [hunk 1]
```diff
@@ -2,11 +2,15 @@ langcode: en
 status: true
 dependencies:
   config:
-    - core.entity_view_mode.media.full
+    - core.entity_view_mode.media.stanford_image_large
+    - core.entity_view_mode.media.stanford_image_medium
+    - core.entity_view_mode.media.stanford_image_stanford_circle
```
Comment (pookmish) [inline config/install/filter.format.stanford_html.yml]: These configs aren't part of this module or stanford_media. they should be removed.

File: config/install/filter.format.stanford_html.yml [hunk 2]
```diff
@@ -82,5 +86,17 @@ filters:
...
     settings:
-      default_view_mode: full
-      allowed_view_modes: {  }
+      default_view_mode: default
+      allowed_media_types: {  }
+      allowed_view_modes:
+        default: default
+        stanford_image_large: stanford_image_large
+        stanford_image_medium: stanford_image_medium
+        stanford_image_stanford_circle: stanford_image_stanford_circle
```
Comment (pookmish) [inline config/install/filter.format.stanford_html.yml]: See above comment

File: composer.json [hunk 3]
```diff
@@ -45,7 +45,7 @@
         "drupal/editor_advanced_link": "^1.4",
         "drupal/embed": "~1.0",
         "drupal/linkit": "~5.0@beta",
-        "su-sws/stanford_media": "^8.2.0@alpha"
+        "su-sws/stanford_media": "dev-8.x-2.x"
```
Comment (sherakama) [inline composer.json]: Revert back before merging.

Comment (pookmish) [reply composer.json]: ugh. more of that.

Comment (sherakama) [reply composer.json]: yeah, WE NEED DEV MODE.

**Comment by sherakama** [comment]
@pookmish maybe we don't need to wait. 8.2.1 has what we need.

**Pull request merged**